libkpod/containerserver.go

```diff
@@ -19,6 +23,7 @@ type ContainerServer struct {
 	ctrNameIndex       *registrar.Registrar
 	ctrIDIndex         *truncindex.TruncIndex
 	imageContext       *types.SystemContext
+	updateLock         *sync.RWMutex
```

Comment:
This can probably be removed - it should only used in the functions we're leaving in the server package